### PR TITLE
Fix: polar label

### PR DESCRIPTION
**Pull request by chenjiandongx**
本次 PR 内容，

1. 修复 Polar 图形 Label 不能正常显示的问题

### CODE REVIEW

File: pyecharts/charts/basic_charts/polar.py [hunk 1]
```diff
@@ -79,4 +80,34 @@ def add(
...
+                    "renderItem": JsCode(
+                        """function(params, api) {
+                        var values = [api.value(0), api.value(1)];
+                        var coord = api.coord(values);
+                        return {
+                            type: 'text',
+                            position: [3 * Math.sin(coord[3]), 3 * Math.cos(coord[3])],
+                            rotation: coord[3] + Math.PI / 2,
+                            origin: [coord[0], coord[1]],
+                            style: {
```
Comment (chfw) [inline pyecharts/charts/basic_charts/polar.py]: hard coded style will become an issue. someone else may want different style.

File: pyecharts/charts/basic_charts/polar.py [hunk 2]
```diff
@@ -79,4 +80,34 @@ def add(
                     "itemStyle": itemstyle_opts,
                 }
             )
+        if label_opts.get("show"):
+            self.options.get("series").append(
```
Comment (chfw) [inline pyecharts/charts/basic_charts/polar.py]: no unit test for this so this code change become a problem for future maintenance.

File: pyecharts/charts/basic_charts/polar.py (hunk 1 above)
Comment (chenjiandongx) [reply pyecharts/charts/basic_charts/polar.py]: Yeap, so i left a `TODO` tag there

File: pyecharts/charts/basic_charts/polar.py (hunk 2 above)
Comment (chenjiandongx) [reply pyecharts/charts/basic_charts/polar.py]: Will perfect it, this pr is not ready for merged

File: pyecharts/charts/basic_charts/polar.py (hunk 1 above)
Comment (chfw) [reply pyecharts/charts/basic_charts/polar.py]: Let’s build quality in.

File: example/page_example.py [hunk 3]
```diff
@@ -140,6 +140,10 @@ def grid_mutil_yaxis() -> Grid:
     )
 
 
-page = Page(layout=Page.SimplePageLayout)
-page.add(bar_datazoom_slider(), line_markpoint(), pie_rosetype(), grid_mutil_yaxis())
-page.render()
+# Page.DraggablePageLayout
```
Comment (chfw) [inline example/page_example.py]: dead code

File: pyecharts/charts/composite_charts/page.py [hunk 4]
```diff
@@ -65,6 +99,39 @@ def _prepare_render(self):
...
                 self.js_dependencies.add(c.theme)
 
+        charts_id = []
+        if self.layout == _MARK_FREEDOM_LAYOUT:
+            self.download_button = True
+            for c in self:
+                charts_id.append("'{}'".format(c.chart_id))
+                self.add_js_funcs(
+                    # make charts resizable and draggable
+                    '$("#%s")'
```
Comment (chfw) [inline pyecharts/charts/composite_charts/page.py]: charts maybe resizeable and draggable but code is not readable.. triple quotes?

Comment (chfw) [reply pyecharts/charts/composite_charts/page.py]: jinja2 can also do templating function

File: pyecharts/datasets/map_filename.json [hunk 5]
```diff
@@ -1,5 +1,8 @@
 {
   "bulma": ["bulma.min", "css"],
+  "jquery": ["jquery.min", "js"],
```
Comment (chfw) [inline pyecharts/datasets/map_filename.json]: hoho, 95% of the web uses jquery. now is 96%

Comment (chenjiandongx) [reply pyecharts/datasets/map_filename.json]: Jquery still in style

File: pyecharts/charts/composite_charts/page.py (hunk 4 above)
Comment (chenjiandongx) [reply pyecharts/charts/composite_charts/page.py]: using python3 f-format now.

File: pyecharts/datasets/__init__.py [hunk 6]
```diff
@@ -1,14 +1,104 @@
...
+import typing
 import urllib.request
 
 import simplejson as json
 
+
+class FuzzyDict(dict):
+    """Provides a dictionary that performs fuzzy lookup"""
+
+    def __init__(self, cutoff: float = 0.6):
```
Comment (chfw) [inline pyecharts/datasets/__init__.py]: missing unit tests.

File: example/page_example.py (hunk 3 above)
Comment (chfw) [reply example/page_example.py]: ping @chenjiandongx

File: test/test_datasets.py [hunk 7]
```diff
@@ -23,3 +23,16 @@ def test_register_url(fake):
                 }
             },
         )
+
+
+def test_fuzzy_search_dict():
```
Comment (chfw) [inline test/test_datasets.py]: OK

**Comment by chfw** [comment]
Remember the dead code!

**Pull request merged**